fast-enrichment-service/proto/enrichment_service.proto

```
@@ -0,0 +1,94 @@
+syntax = "proto3";
+
+package gpp.g3.enrichment;
```

Buf (1.54.0)

Files with package "gpp.g3.enrichment" must be within a directory "gpp/g3/enrichment" relative to root but were in directory "fast-enrichment-service/proto".

(PACKAGE_DIRECTORY_MATCH)